plugins/modules/make.py

```diff
@@ -52,9 +52,10 @@
     type: dict
   target:
     description:
-      - The target to run.
+      - The targets to run.
       - Typically this would be something like V(install), V(test), or V(all).
-    type: str
+    type: list
+    elements: str
```

Comment:
This is technically a breaking change, as the module now behaves differently when a string is provided to `target` that contains a comma.